### PR TITLE
Translate type system almost done with some TODO.

### PR DESCRIPTION
**Pull request by DataCorrupted**
#63

### CODE REVIEW

File: docs/en/src/context.md [hunk 1]
```diff
@@ -1 +1,21 @@
 # Context
+
+The main goal of `Context` is to acquire global data attached to Schema. **Note that if the return value of your Resulve boored data in `Context`, you need to explictly state the lifetime of the argument.**
```
Comment (IcanDivideBy0) [inline docs/en/src/context.md:3]: Note that if the return value of your resolve function is borrowed from `Context`, you need to explicitly state the lifetime of the argument.

File: docs/en/src/define_enum.md [hunk 2]
```diff
@@ -1 +1,22 @@
 # Enum
+
+It's easy to define an `Enum`, here we have an example:
+
+**Async-graphql can automatically change the name of each item to GraphQL's convsion (all capitals and underscores), you can also use `name` to rename.**
```
Comment (IcanDivideBy0) [inline docs/en/src/define_enum.md:5]: GraphQL's **convension**

I believe that's named `CONSTANT_CASE`

File: docs/en/src/define_input_object.md [hunk 3]
```diff
@@ -1 +1,31 @@
 # InputObject
+
+<!--Input Object and SimpleObject inconsistant space.-->
+You can define an `Object` as argument, GraphQL calls it `InputObject`.
+The definition of `InputObject` is similar to [SimpleObject](define_simple_object.md).
+However, `InputObject` can only be used for output and `SimpleObject` can only be used as input.
```
Comment (IcanDivideBy0) [inline docs/en/src/define_input_object.md:6]: `InputObject` and `SimpleObject` are swaped

**Comment by DataCorrupted** [comment]
Thx. I just fixed all typo.

**Comment by sunli829** [comment]
Thank you for using your precious time to help me do these tasks, and they have done so well. If you have any questions about the use of this library later, please let me know and I will do my best to help you. 😁

**Comment by sunli829** [comment]
@IcanDivideBy0 Also thank you for your review. 😁

**Pull request merged**